app/vep/utils/nextflow.py

```diff
@@ -24,9 +24,28 @@ def launch_workflow(pipeline_params: PipelineParams):
             SEQERA_WORKFLOW_LAUNCH_URL, headers=headers, json=payload
         )
         response.raise_for_status()
-        responseJson = response.json()
-        return responseJson["workflowId"]
+        response_json = response.json()
+        return response_json["workflowId"]
 
     except Exception as e:
         logger.exception("VEP WORKFLOW SUBMISSION ERROR: ", e)
-        return None
+        raise Exception
+
+
+async def get_workflow_status(job_id):
+    try:
+        _headers = {
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {NF_TOKEN}",
+        }
+        _seqera_workflow_status_url = f"{SEQERA_API}/workflow/{job_id}"
+        response = requests.get(_seqera_workflow_status_url, headers=_headers)
+
+        response.raise_for_status()
+        pipeline_status = PipelineStatus(job_id=job_id, status=response.json())
+        return pipeline_status
+    except HTTPError as e:
+        raise e
+    except Exception as e:
+        logger.exception("VEP connection error: ", e)
```

Comment:
If the purpose here is to log the connection issue while requesting seqera API then it would be good to be explicit and say connection issue with seqera API.

Reply:
It's a bit more general than that because we're not connecting to seqera here. It can be any unexpected issue that happened while connecting to this API exactly like line 31.